crawler.py

```diff
@@ -0,0 +1,91 @@
+""" Async Web Crawler """
+
+import asyncio
+import html
+import requests
+from re import findall, compile
+from requests_html import AsyncHTMLSession
+from typing import List
```

Comment:
Optimize imports. You have unused imports here also. Ctrl + Alt + O is the keyboard shortcut.